dev/docker/eth-dev.yml

```diff
@@ -0,0 +1,37 @@
+version: '3.2'
+
+# creates a nucypher node and a ganache server which can be used for
+# blockchain simulations
+
+# docker-compose -f eth-dev.yml up
+
+# in another shell
+# docker-compose -f eth-dev.yml run nucypher-dev nucypher-deploy contracts --poa --provider-uri http://ganache:7545
+# docker-compose -f eth-dev.yml run nucypher-dev nucypher felix init --checksum-address "<from output>" --network devnet
+# docker-compose -f eth-dev.yml run  nucypher-dev nucypher felix createdb --checksum-address "<from output>" --network devnet
+# docker-compose -f eth-dev.yml run --service-ports nucypher-dev nucypher felix run --checksum-address "<from output>" --network devnet
+
+services:
+  nucypher-dev:
+    ports:
+      - "6151:6151"
+    volumes:
+      - ../..:/code
+    build:
+      context: ../..
+      dockerfile: dev/docker/Dockerfile
+    image: dev:nucypher
+    container_name: nucypher-dev
+    environment:
+      - NUCYPHER_CONFIG_ROOT=/code/config-data
+      - NUCYPHER_FELIX_DB_SECRET=1h918hd98327hf8723f8723f872g38f7g
```

Comment:
Can this be read via local envvar? Also be sure this secret is destroyed.

Reply:
Doesn't docker have a secrets manager that can be used for just this thing?

Reply:
I don't understand why this matters.  I could just put "non-secret-dev-string" here. Wouldn't that be the same thing?  It's only for local dev/testing.

Reply:
Well - It's not just for dev, is it - we may use this (or someone else may) in production? Also It's ambiguous that this is indented to be a development password.  Perhaps `"this-is-not-a-secure-password` is more clear.

Reply:
@vepkenez Yeah, let's make this `dont-use-this-password-in-prod` or something so it's clear to people we aren't using this secret haha.